apps/dashboard/config/application.rb

```diff
@@ -43,5 +43,9 @@ class Application < Rails::Application
       config.paths["config/initializers"] << ::Configuration.config_root.join("initializers").to_s
       config.paths["app/views"].unshift ::Configuration.config_root.join("views").to_s
     end
+
+    if Rails.env.test?
+      config.paths["app/views"].unshift "test/fixtures/config/views"
+    end
```

Comment:
I tried every way to Sunday to fit this into `environments/test.rb` but was unable to. I don't really like putting it here, but I was unable to get rails to reload the paths in the `test.rb` file.

Reply:
What about test/test_helper.rb?

Reply:
Yep that worked!